libs/perun/directives/src/lib/middle-click-router-link.ts

```diff
@@ -1,20 +1,48 @@
-import { Directive, HostListener, Input } from '@angular/core';
+import { Directive, ElementRef, HostListener, Input, Renderer2 } from '@angular/core';
 
 @Directive({
   selector: '[perunWebAppsMiddleClickRouterLink]',
 })
 export class MiddleClickRouterLinkDirective {
   @Input() perunWebAppsMiddleClickRouterLink: string[];
 
+  constructor(private ref: ElementRef, private renderer: Renderer2) {
+    const htmlElement: HTMLElement = this.ref.nativeElement as HTMLElement;
+    this.renderer.listen(htmlElement, 'click', (event: MouseEvent) => {
+      // ctrl (command) + left mouse click will stop routing and open router link in the new tab
+      if (
+        // ctrlKey = support for Windows/Linux keyboards
+        // metaKey = support for Mac keyboards
+        (event.ctrlKey || event.metaKey) &&
+        event.button === 0 &&
+        this.perunWebAppsMiddleClickRouterLink !== null &&
+        this.perunWebAppsMiddleClickRouterLink !== undefined
+      ) {
```

Comment:
Nitpick, for both occurences.
```suggestion
      if (
        // ctrlKey = support for Windows/Linux keyboards
        // metaKey = support for Mac keyboards
        (event.ctrlKey || event.metaKey) &&
        event.button === 0 &&
        this.perunWebAppsMiddleClickRouterLink
      ) {
```